R/utils.R

```diff
@@ -64,7 +64,7 @@ source_files = function(format = NULL, config = load_config(), all = FALSE) {
     recursive = subdir_yes, full.names = subdir_yes
   )
   if (length(config[['rmd_files']]) > 0) {
-    files = config[['rmd_files']]
+    files = c(config[['rmd_files']], files)
```

Comment:
The concatenation needs to be done after the next `if` (i.e. after the list is resolved to a character vector).